ui/framework/src/ui-framework/timeline/SolarTimelineDataProvider.ts

```diff
@@ -85,6 +70,5 @@ export class SolarTimelineDataProvider extends BaseSolarDataProvider {
 
     newSettings.color = color.tbgr;
     displayStyle.settings.solarShadows = SolarShadowSettings.fromJSON(newSettings);
-    this._viewport.invalidateScene();

```

Comment:
@bsteinbk I removed the `invalidateScene` because the Viewport now automatically detects changes to the shadow settings.